From 519612d0e2d240bb094fa80190daf7dccecf2882 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 19 Oct 2016 12:34:41 -0700
Subject: [PATCH] IMGAPI-596 update imgapi and docker to use
 restify-clients@1.4.0 with upstreamed http_proxy support (v3.2.3) Reviewed
 by: Todd Whiteman <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 CHANGES.md   | 7 ++++++-
 package.json | 2 +-
 2 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 5fb41ad..00fe6b0 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,6 +1,11 @@
 # node-docker-registry-client Changelog
 
-## 3.2.3 (not yet released)
+## not yet released
+
+## 3.2.3
+
+- IMGAPI-596 Move from restify-clients fork to restify-clients@1.4.0 with
+  upstreamed HTTP proxy support.
 
 ## 3.2.2
 
diff --git a/package.json b/package.json
index 4fe70df..fc4db8c 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "docker-registry-client",
-    "version": "3.2.2",
+    "version": "3.2.3",
     "description": "node.js client for the Docker Registry API",
     "author": "Joyent (joyent.com)",
     "main": "./lib/index.js",
-- 
2.21.0

